querybook/server/env.py

```diff
@@ -83,6 +83,8 @@ class QuerybookSettings(object):
     STORE_MAX_UPLOAD_CHUNK_NUM = int(get_env_config("STORE_MAX_UPLOAD_CHUNK_NUM"))
     STORE_MAX_READ_SIZE = int(get_env_config("STORE_MAX_READ_SIZE"))
     STORE_READ_SIZE = int(get_env_config("STORE_READ_SIZE"))
+    STORE_BUCKET_S3V4_ENABLED = get_env_config("STORE_BUCKET_S3V4_ENABLED")
+    STORE_BUCKET_REGION = get_env_config("STORE_BUCKET_REGION")
```

Comment:
I am wondering if it would be better if we make this into a more general setting called `AWS_REGION` which can be  used for elasticsearch as well.
Once you made that change, can you update all `us-east-1` with this config value instead?

Reply:
Made the changes. Please review.